graphql-server/src/queryFactory/dolt/queries.ts

```diff
@@ -84,6 +84,9 @@ export const callPullRemote = `CALL DOLT_PULL(?, ?)`;
 
 export const callPushRemote = `CALL DOLT_PUSH(?, ?)`;
 
+export const callFetchRemote = (branchName?: string) =>
+  branchName ? `CALL DOLT_FETCH(?, '${branchName}')` : `CALL DOLT_FETCH(?)`;
```

Comment:
Should use prepared statements instead of string interpolation for queries whenever possible
```suggestion
export const callFetchRemote = (hasBranchName?: boolean) =>
  `CALL DOLT_FETCH(?${hasBranchName ? ", ?" : ""})`
```